### ISSUE TITLE
Take words which are not required and add a few floating point words to ROM

### ISSUE THREAD

**Issue by rachel8973** [rachel8973/RC2014-FORTH#2]
From what I understand there a few words involved with handling "SCREENS" of words which was a way of building, editing and saving words.  Thinking of pruning these out and seeing if there is room for a few basic floating point words i.e. F. F+ F- F* F/ and the detecting input of FP numbers from the existence of a decimal point.  Looking at the Jupiter Ace Rom source for pointers.

**Comment by larsbrinkhoff** [comment]
Right.  "Screens", or "blocks", are only useful to interface with a mass storage device.

Here is a description of the optional block wordset:  
http://lars.nocrew.org/dpans/dpans7.htm

**Comment by rachel8973** [comment]
Thanks lars
